File: ML/rachitavya_RachitavyaSharma_2125cs1053_2/GettingStarted/countdigits.java
import java.util.*;
  
  public class countdigits{
  
  public static void main(String[] args) {
    // write your code here 
    Scanner sc=new Scanner(System.in);
    int num=sc.nextInt();
    int c;
    for(c=0;num>0;num/=10)
    c++;
    System.out.print(c);
    sc.close();
   }
  }

File: Web/kashish12_kashishalikhan_2125cse1133_02/GettingStarted/countdigits.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    int p,count = 0;
    while(n != 0){
        p = n % 10;
        count ++;
        n = n / 10;
    }
    cout<<count;
    return 0;
    
    //write your code here
    
}